javav2/example_code/ses/metadata.yaml

```diff
@@ -0,0 +1,11 @@
+--- 
+files:
+  - path: src/main/java/aws/example/ses/SendMessage.java
```

Comment:
Shouldn't the path be src/main/javav2?

Reply:
The path under main is java. This is correct now.